### ISSUE TITLE
cosmic-ray init causes cosmic-ray baseline to fail

### ISSUE THREAD

**Issue by rob-smallshire** [sixty-north/cosmic-ray#275]
Using the latest version from git e6ba9d32e2065ed1dd8cf6004bcfa6f0dbf0c1cc , having also installed the py.test runner, I was trying to get up-and-running, but cosmic-ray init failed to run properly:

```
$ cosmic-ray init cosmic-ray-pytest-local2.conf session1
[?] Top-level module: asm68
Test runner
  (0) unittest
  (1) pytest
[?] Enter menu selection: 1
[?] Test args: 
Execution engine
  (0) local
[?] Enter menu selection: 0
baseline failed
Traceback (most recent call last):
  File "/Users/rjs/.virtualenvs/asm68/lib/python3.6/site-packages/cosmic_ray/testing/test_runner.py", line 55, in __call__
    test_result = self._run()
  File "/Users/rjs/.virtualenvs/asm68/lib/python3.6/site-packages/cosmic_ray_pytest_runner/runner.py", line 31, in _run
    pytest.main(self.test_args.split(),
AttributeError: 'NoneType' object has no attribute 'split'

Traceback (most recent call last):
  File "/Users/rjs/.virtualenvs/asm68/bin/cosmic-ray", line 11, in <module>
    sys.exit(main())
  File "/Users/rjs/.virtualenvs/asm68/lib/python3.6/site-packages/cosmic_ray/cli.py", line 298, in main
    common_option_handler=common_option_handler)
  File "/Users/rjs/.virtualenvs/asm68/lib/python3.6/site-packages/docopt_subcommands/__init__.py", line 69, in main
    result = commands(argv)
  File "/Users/rjs/.virtualenvs/asm68/lib/python3.6/site-packages/docopt_subcommands/subcommands.py", line 134, in __call__
    return handler(config)
  File "/Users/rjs/.virtualenvs/asm68/lib/python3.6/site-packages/cosmic_ray/cli.py", line 112, in handle_init
    subprocess.check_call(command.split())
  File "/usr/local/var/pyenv/versions/3.6.2/lib/python3.6/subprocess.py", line 291, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['cosmic-ray', 'baseline', 'cosmic-ray-pytest-local2.conf']' returned non-zero exit status 2.
```

**Comment by rob-smallshire** [comment]
To be clear, I have

1. Cloned the git repo.
2. Used `python setup.py sdist` to build a source distribution of cosmic-ray, and again to build a source distribution of the py.test runner.
3. Used `pip` to install the aforementioned source distributions into the virtualenv of the project under test.
4. Invoked `cosmic-ray init` as above.

**Comment by abingham** [comment]
The problem is that if you don't put anything for "Test args:" you get `None` as the test args when running tests (including the baseline). We should probably force people to put something, or at least give better guidance. 

The whole menu-driven approach to building configs is experimental at best, and maybe it's going to cause more trouble than its worth.

**Comment by rob-smallshire** [comment]
Okay. At this point I had no idea what test args was – neither the Quickstart nor the tools say what is required, although I see that careful inspection of the example config.yml in the Quickstart reveals the answer.

**Comment by rob-smallshire** [comment]
I'm going to close this one, and open a new one about the lack of instruction for test args.

**Comment by rob-smallshire** [comment]
I'm going to reopen this on the basis that the malformed config file should give a helpful error message, rather than a stack trace.

**Comment by abingham** [comment]
@rob-smallshire The initial issue reported here is resolved by PR #311. Your point about helpful messages still stands, though. 

Do you think the issue of message is adequately covered by #283? If so, let's close this out.

**Comment by rob-smallshire** [comment]
Yes, I agree that #283 is now a better description of the remaining issue.
